game/menu/gamedisplay.gd

```diff
@@ -2,11 +2,15 @@ extends MarginContainer
 
 signal pressed(game_file)
 
-var game_file: ConfigFile
+class_name GameDisplay
 
+var game_file: ConfigFile
+var loadbtn: Button
+var loadbtn2: Button
```

Comment:
Variables should be in snake_case.
Candidates for private variables?

Reply:
And maybe more meaningful names, like load_popup_button or something.

Reply:
Sure, those can be private and we can make them more meaningful.